rustv1/examples/ec2/src/ec2.rs

```diff
@@ -0,0 +1,530 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{net::Ipv4Addr, time::Duration};
+
+use aws_sdk_ec2::{
+    client::Waiters,
+    error::ProvideErrorMetadata,
+    operation::{
+        allocate_address::AllocateAddressOutput, associate_address::AssociateAddressOutput,
+    },
+    types::{
+        DomainType, Filter, Image, Instance, InstanceType, IpPermission, IpRange, KeyPairInfo,
+        SecurityGroup, Tag,
+    },
+    Client as EC2Client,
+};
+use aws_sdk_ssm::types::Parameter;
+use aws_smithy_runtime_api::client::waiters::error::WaiterError;
+
+// #[cfg(test)]
+// use mockall::automock;
```

Comment:
Could be removed?

Reply:
Uncommented; it's a rust-analyzer thing where it defaults to [test] when editing, so it sees the mock, not the impl, when trying to develop.